templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -167,6 +184,36 @@ objects:
             secret:
               secretName: asb-auth-secret
 
+- apiVersion: v1
```

Comment:
I think before we merge we should get this working as a container inside the asb pod IMO

Reply:
My belief is that this is so we don't have to expose a public facing route in order for the broker to be able to talk to it. Is that right?

Reply:
@djzager that's is true, but the broker in this case never talks to the redirector. It's for external clients (OpenShift UI).